c-api/mapping.po

```diff
@@ -6,105 +6,150 @@ msgstr ""
 "Project-Id-Version: Python 3.6\n"
 "Report-Msgid-Bugs-To: \n"
 "POT-Creation-Date: 2018-12-21 09:48+0100\n"
-"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
-"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"PO-Revision-Date: 2019-11-08 18:39+0100\n"
 "Language-Team: FRENCH <traductions@lists.afpy.org>\n"
 "Language: fr\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
+"Last-Translator: \n"
+"X-Generator: Poedit 2.2.4\n"
 
 #: ../Doc/c-api/mapping.rst:6
 msgid "Mapping Protocol"
-msgstr ""
+msgstr "Protocole de correspondance"
 
 #: ../Doc/c-api/mapping.rst:8
 msgid ""
 "See also :c:func:`PyObject_GetItem`, :c:func:`PyObject_SetItem` and :c:func:"
 "`PyObject_DelItem`."
 msgstr ""
+"Voir aussi c:func:`PyObject_GetItem`, :c:func:`PyObject_SetItem` et :c:func:"
+"`PyObject_DelItem`."
 
+# - " in general case it is impossible to determine what the type of keys it supports."
+# → pas correct en VO !
+# - "This function always succeeds." équivaut-il à "jamais d'erreur" ?
 #: ../Doc/c-api/mapping.rst:14
 msgid ""
 "Return ``1`` if the object provides mapping protocol or supports slicing, "
 "and ``0`` otherwise.  Note that it returns ``1`` for Python classes with a :"
 "meth:`__getitem__` method since in general case it is impossible to "
 "determine what the type of keys it supports.  This function always succeeds."
 msgstr ""
+"Renvoie ``1`` si l'objet supporte le protocole de correspondance ou "
+"l'indexation, et ``0`` sinon. Notez qu'elle renvoie ``1`` pour les classes "
+"Python avec un méthode :meth:`__getitem__` puisque, dans le cas général, il "
+"est impossible de déterminer quel type de clef est supporté. Cette fonction "
+"ne provoque jamais d'erreur."
 
+# "C'est l'équivalent de" est moche, mais comme j'ai pas de précision sur "This"...
 #: ../Doc/c-api/mapping.rst:26
 msgid ""
 "Returns the number of keys in object *o* on success, and ``-1`` on failure. "
 "This is equivalent to the Python expression ``len(o)``."
 msgstr ""
+"Renvoie le nombre de clefs dans l'objet *o*, et ``-1`` en cas d'échec. C'est "
+"l'équivalent de l'expression Python ``len(o)``."
 
+# - "Return element" est incorrect en anglais.
+# On renvoie le premier ? Un élément quelconque qui correspond ? Tous ?
+# - Cf. le commentaire précédent à propos de "this".
+# - Je suppose que *key* est défini par ailleurs, donc je l'ai laissé tel quel.
```

Comment:
Quand tu as des doutes, lis la doc !

https://docs.python.org/3/c-api/mapping.html

Reply:
*Of course*, RTFM ! Bon là c'est heureux, il fallait effectivement laisser *key* telle quelle.